smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -38,75 +38,66 @@ contract MixinTransfer is
   uint public transferFeeBasisPoints;
 
   /**
-  * @notice Allows the key owner to safely share their key (parent key) by
-  * transferring a portion of the remaining time to a new key (child key).
-  * @param _to The recipient of the shared key
-  * @param _tokenId the key to share
+  * @notice Allows the key owner to safely transfer a portion of the remaining time 
+  * from their key to a new keyx
+  * @param _tokenIdFrom the key to share
+  * @param _to The recipient of the shared time
   * @param _timeShared The amount of time shared
   */
   function shareKey(
```

Comment:
As I am thinking about it we need a "mergeKeyInto" function that takes the time from a key and assigns to another key. Because here I cannot share key with someone who already has a key and add them time.

Reply:
yes we need a `mergeKeys(uint _tokenIdFrom, uint _tokenIdTo)` that will destroy the from key and add remaining time to the new one. Will do in another PR - I think will add `burn` first so we can make use of it